db/schema.rb

```diff
@@ -47,6 +47,28 @@
     t.integer  "proposal_id"
   end
 
+  create_table "cart_item_traits", force: true do |t|
+    t.text     "name"
+    t.text     "value"
+    t.integer  "cart_item_id"
+    t.datetime "created_at"
+    t.datetime "updated_at"
+  end
+
+  create_table "cart_items", force: true do |t|
+    t.string   "vendor"
+    t.text     "description"
+    t.string   "url"
+    t.text     "notes"
+    t.integer  "quantity"
+    t.text     "details"
+    t.string   "part_number"
+    t.float    "price"
+    t.integer  "cart_id"
+    t.datetime "created_at"
+    t.datetime "updated_at"
+  end
```

Comment:
Try merging master and migrating the db. I don't think `cart_items` and `cart_items_traits` are still used.